site/en/docs/devtools/lighthouse/index.md

```diff
@@ -47,164 +57,111 @@ has 2 important functions:
-    code.
-5.  Click **Show Live**. The demo opens in a new tab. This tab will be referred to as the _demo
-    tab_. It may take a while for the site to load.
+1. Open the [source of this demo website][2] and click **Remix** in the upper right corner. A new project opens in a tab. This tab will be referred to as the _editor tab_.
 
-    {% Img src="image/admin/2kZXAZgIbvKrafszL6X3.png", alt="The demo tab.", width="800", height="588" %}
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/vNL328lOu6idOm4AKNdP.png", alt="The original source and the editor tab after clicking Remix.", width="800", height="466" %}
 
-    **Figure 4**. The demo tab
+   The name of the project changes from **tony** to some randomly-generated name. You now have your own editable copy of the code. Later on, you'll make changes to this code.
 
-6.  Press Command+Option+J (Mac) Control+Shift+J (Windows, Linux, ChromeOS). Chrome DevTools opens
-    up alongside the demo.
+1. At the bottom of the editor tab, click **Preview** > **Preview in a new window**. The demo opens in a new tab. This tab will be referred to as the _demo tab_. It may take a while for the site to load.
 
-    {% Img src="image/admin/QyoSneXNo8Npiy28nJlU.png", alt="DevTools and the demo.", width="800", height="476" %}
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/dX6kVyYogDqrDVcM4WzJ.png", alt="The demo tab.", width="800", height="614" %}
 
-    **Figure 5**. DevTools and the demo
+1. [Open DevTools](/docs/devtools/open) alongside the demo.
 
-For the rest of the screenshots in this tutorial, DevTools will be shown as a separate window. You
-can do this by pressing Command+Shift+P (Mac) or Control+Shift+P (Windows, Linux, ChromeOS) to open
-the Command Menu, typing `Undock`, and then selecting **Undock into separate window**.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/IgZOUlF1RxARqEGoKjKF.png", alt="DevTools and the demo.", width="800", height="535" %}
 
-{% Img src="image/admin/YgPOzT6Re0aIGezSKPZr.png", alt="Undocked DevTools.", width="800", height="514" %}
-
-**Figure 6**. Undocked DevTools
+{% Aside 'important' %}
+For the rest of the screenshots in this tutorial, DevTools is shown in a [separate window](/docs/devtools/customize/#placement).
+{% endAside %}
 
 ### Establish a baseline {: #baseline }
 
 The baseline is a record of how the site performed before you made any performance improvements.
 
-1.  Click the **Audits** tab. It may be hidden behind the **More Panels**
-    {% Img src="image/admin/IxLXhwoZksgzzdzr9OCx.png", alt="More Panels", width="18", height="16" %} button. There's a
-    Lighthouse on this panel because the project that powers the Audits panel is called
-    [Lighthouse][3].
+1.  Open the **Lighthouse** panel. It may be hidden behind {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/V93Xxk8HvmByCBGzMdW4.svg", alt="More panels.", width="20", height="20" %} **More panels**. The panel is powered by [**Lighthouse**](/docs/lighthouse).
```

Comment:
Did we explain the Lighthouse panel vs Lighthouse different somewhere? It is because this sentence might look confusing.